pkg/operator/hive/hive.go

```diff
@@ -517,3 +518,47 @@ func (r *ReconcileHiveConfig) deleteAllSyncSetInstances(hLog log.FieldLogger) (n
 	}
 	return
 }
+
+func (r *ReconcileHiveConfig) deployMonitoring(hLog log.FieldLogger, h resource.Helper, instance *hivev1.HiveConfig) error {
+	hiveNSName := getHiveNamespace(instance)
+
+	if !instance.Spec.ExportMetrics {
+		hLog.Info("metrics were not requested to be extracted, so skipping")
+		return nil
+	}
```

Comment:
Here's where we need to _undo_ these changes if previously applied, per HIVE-1655.